go.mod

```diff
@@ -2,4 +2,4 @@ module github.com/kubernetes-sigs/depstat
 
 go 1.16
 
-require github.com/spf13/cobra v1.1.1
+require github.com/spf13/cobra v1.1.3
```

Comment:
Refer to [this](https://github.com/kubernetes-sigs/depstat/pull/20#discussion_r621839266).